database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -71,7 +71,7 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
       )
     } yield workflowStoreEntries
 
-    runTransaction(action)
+    runTransaction(action, TransactionIsolation.ReadCommitted)
```

Comment:
It would be nice to briefly call out why it's appropriate to override the default in these two cases

Reply:
Three cases, but actually I don't know why this is okay. Allowing non-repeatable read isolation in the workflow pickup in particular seems kind of dicey. Calling for backup @kcibul

Reply:
Not a merge blocker by any means FWIW

Reply:
Updates based on Reading The Friendly MySQL docs. I removed `ReadCommitted` from two of the call sites since I don't think it was having any effect there: the abort scan is read-only and therefore non-locking, while the heartbeat writes are index-driven and should not have been affected by the change from `RepeatableRead` since only the updated rows were being locked anyway. The remaining transaction should be affected by the `ReadCommitted` change, hopefully my argument for its safety is plausible.